### PR TITLE
Onboard lift shift ebook

### PR DESCRIPTION
**Pull request by BillWagner**
This is another eBook import.

This one is for lift and shift scenarios.

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and re-opening to force a new build.

File: docs/standard/modernize-with-azure-and-containers/index.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+At the first two migration levels, you can just lift and shift your applications:
+
+**Level 1: Cloud Infrastructure-Ready**: In this migration approach, you simply rehost or move your current on-premises applications to an infrastructure as a service ([IaaS](https://azure.microsoft.com/overview/what-is-iaas/)) platform. Your apps have almost the same composition as before, but now you deploy them to VMs in the cloud.
+
+**Level 2: Cloud DevOps-Ready**: At this level, after an initial lift and shift, and still without re-architecting or altering code, you can gain even more benefits from running your app in the cloud. You improve the agility of your applications to ship faster by refining your enterprise development operations (DevOps) processes. You achieve this by using technologies like Windows Containers, which is based on Docker Engine. Containers remove the friction that's caused by application dependencies when you deploy in multiple stages. Containers also use additional cloud managed services related to data, monitoring, and continuous integration/continuous deployment (CI/CD) pipelines.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/index.md]: technologies like Windows Containers, which is based... --> a technology like Windows Containers, which is based

File: docs/standard/modernize-with-azure-and-containers/index.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+The third level of maturity is the ultimate goal in the cloud, but it's optional for many apps and not the main focus of this guide:
+
+**Level 3: Cloud-Optimized**: This migration approach typically is driven by business need, and targets modernizing your mission-critical applications. At this level, you use PaaS services to move your apps to PaaS computing platforms. You implement [cloud-native](https://www.gartner.com/doc/3181919/architect-design-cloudnative-applications) applications and microservices architecture to evolve applications with long-term agility, and to scale to new limits. This type of modernization usually requires architecting specifically for the cloud. New code often must be written, especially when you move to cloud-native application and microservice-based models. This approach can help you gain benefits that are difficult to achieve in your monolithic and on-premises application environment.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/index.md]: no comma before "and targets"

File: docs/standard/modernize-with-azure-and-containers/index.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+> | **Challenges** |
+> | <li> Smaller cloud value, other than shift in operating expense or closing datacenters <li> Very little is managed: No OS or middleware patching; might infrastructure solutions, like Terraform, Spinnaker, or Puppet | <li> Containerizing is an additional step in the learning curve require immutable | <li> Might require significant code refactoring or rewriting (increased time and budget) |
+>> **Table 1-1.** Benefits and challenges of modernization paths for existing .NET applications and services
+
+### Key technologies and architectures by maturity level
+
+.NET Framework applications initially started with the .NET Framework version 1.0, which was released in late 2001. Then, companies moved towards newer versions (such as 2.0, 3.5 and .NET 4.x). Most of those applications run on Windows Server and Internet Information Server (IIS), and used a relational database, like SQL Server, Oracle, MySQL or any other RDBMS.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/index.md]: either run --> ran, or used --> use; the tenses need to agree

File: docs/standard/modernize-with-azure-and-containers/index.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+Most existing .NET applications might nowadays be based on .NET Framework 4.x, or even on .NET Framework 3.5, and use web frameworks like ASP.NET MVC, ASP.NET Web Forms, ASP.NET Web API, Windows Communication Foundation (WCF), ASP.NET SignalR, and ASP.NET Web Pages. These established .NET Framework technologies depend on Windows. That dependency is important to consider if you are simply migrating legacy apps, and you want to make minimal changes to your application infrastructure.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/index.md]: nowadays? 
no comma after "legacy apps"

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**